### PR TITLE
Added a FAQ section to azure client provider page

### PR DESCRIPTION
**Pull request by petery789**
# Writer's Quality Checklist

Before merging your PR, did you:

- [x] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/configure-client-management-azure.adoc [hunk 1]
```diff
@@ -63,6 +63,17 @@ For a list of supported grant types, see the https://docs.microsoft.com/en-us/az
...
+== Frequently Asked Questions
+
+
+**1. Is https://docs.mulesoft.com/policies/policies-included-openid-token-enforcement#token-validation-endpoint-authentication[Token Validation] supported when I use Azure AD as a client provider?**
+
+Yes. Although Microsoft Azure AD doesn't have an introspect endpoint out of the box, Anypoint Platform makes its own implementation on token introspection. Note, be sure to follow https://docs.mulesoft.com/access-management/configure-client-management-azure#oauth-2-0-endpoints[Oauth2.0 Endpoints] section to generate supported tokens. 
+
+**2. When attempting to enforce the OIDC policy by invoking an Anypoint API using a valid token obtained from Azure AD, I am getting a "Token has been revoked" error. I have verified the token validity, how can I troubleshoot this?**
+
+Please first refer to OAuth 2.0 Endpoints (https://docs.mulesoft.com/access-management/configure-client-management-azure#oauth-2-0-endpoints) section to confirm that you have obtained the token via the supported approaches. Then, if it still fails the token validation, please contact Engineering team.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/configure-client-management-azure.adoc]:
```suggestion

== Token Validation

Although Microsoft Azure AD does not provide an introspection endpoint out of the box, Anypoint Platform has a xref:policies::policies-included-openid-token-enforcement.adoc#token-validation-endpoint-authentication[policy that implements token introspection for token validation] when you use Azure AD as a client provider.

To use this policy, follow the documentation on xref:configure-client-management-azure.adoc#oauth-2-0-endpoints[OAuth2.0 Endpoints] to generate supported tokens. 

=== Token has been Revoked Error 

When you attempt to enforce the OIDC policy by invoking an Anypoint API using a valid token obtained from Azure AD and get a `Token has been revoked` error, refer to the xref:configure-client-management-azure.adoc#oauth-2-0-endpoints[documentation on OAuth 2.0 endpoints] to confirm that you have obtained the token using a supported method. If the token validation still fails, please contact the Engineering team. 
```

Comment (sarathecoubian) [reply modules/ROOT/pages/configure-client-management-azure.adoc]: Who is "the engineering team" in this context? Do we not mean Support? Or do we want the IAM team? @petery789 👀

Comment (petery789) [reply modules/ROOT/pages/configure-client-management-azure.adoc]: Good question! Probably support. I suppose support should be the first to contact; users don't directly contact our team.

File: modules/ROOT/pages/configure-client-management-azure.adoc [hunk 2]
```diff
@@ -63,6 +63,17 @@ For a list of supported grant types, see the https://docs.microsoft.com/en-us/az
...
+
+== Token Validation
+
+Although Microsoft Azure AD does not provide an introspection endpoint out of the box, Anypoint Platform has a xref:policies::policies-included-openid-token-enforcement.adoc#token-validation-endpoint-authentication[policy that implements token introspection for token validation] when you use Azure AD as a client provider.
+
+To use this policy, follow the documentation on xref:configure-client-management-azure.adoc#oauth-2-0-endpoints[OAuth2.0 Endpoints] to generate supported tokens. 
+
+=== Token has been Revoked Error 
+
+When you attempt to enforce the OIDC policy by invoking an Anypoint API using a valid token obtained from Azure AD and get a `Token has been revoked` error, refer to the xref:configure-client-management-azure.adoc#oauth-2-0-endpoints[documentation on OAuth 2.0 endpoints] to confirm that you have obtained the token using a supported method. If the token validation still fails, please contact the Engineering team.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/configure-client-management-azure.adoc:75]:
```suggestion

== Token Validation

Although Microsoft Azure AD does not provide an introspection endpoint out of the box, Anypoint Platform has a xref:policies::policies-included-openid-token-enforcement.adoc#token-validation-endpoint-authentication[policy that implements token introspection for token validation] when you use Azure AD as a client provider.

To use this policy, follow the documentation on xref:configure-client-management-azure.adoc#oauth-2-0-endpoints[OAuth2.0 Endpoints] to generate supported tokens. 

=== Token has been Revoked Error 

When you attempt to enforce the OIDC policy by invoking an Anypoint API using a valid token obtained from Azure AD and get a `Token has been revoked` error, refer to the xref:configure-client-management-azure.adoc#oauth-2-0-endpoints[documentation on OAuth 2.0 endpoints] to confirm that you have obtained the token using a supported method. If the token validation still fails, contact Customer Support. 
```

Comment (sarathecoubian) [inline modules/ROOT/pages/configure-client-management-azure.adoc]:
```suggestion
When you attempt to enforce the OIDC policy by invoking an Anypoint API using a valid token obtained from Azure AD and get a `Token has been revoked` error, refer to the xref:configure-client-management-azure.adoc#oauth-2-0-endpoints[documentation on OAuth 2.0 endpoints] to confirm that you have obtained the token using a supported method. If the token validation still fails, contact Support. 
```

**Review by sarathecoubian — APPROVED** [review]

**Pull request merged**